programs/rc-fall.html

```diff
@@ -6,27 +6,110 @@
 
 
   
-  <!-- How It Works row -->
-  <div class="row text-center">
-    <h2> A Two-Part Event </h2>
-    <div class="col-md-6">
-      <div class="notecard" style="float:right">
-      <div class="big-txt"> <i class="fa fa-cogs"></i> </div>
-      <h3>RC: Reboot</h3>
+<div class="container-fluid">
+  <ul class="nav nav-tabs">
+    <li class="active"><a href="#about" data-toggle="tab" aria-expanded="false">About</a>
+    <li class=""><a href="#external" data-toggle="tab" aria-expanded="false">External Information</a></li>
+    <li class=""><a href="#internal" data-toggle="tab" aria-expanded="false">Internal</a></li>
```

Comment:
Last Spring, one of the things that Tommy told me to remove from the Kickoff public page was the link to the staff internal page.  His reasoning was that he believed that we should blackbox internal PiE operations from the public.  Just something to consider